security/uacme/Portfile

```diff
@@ -0,0 +1,55 @@
+PortSystem          1.0
+PortGroup           github 1.0
+
+name                uacme
+version             1.7.4
+revision            0
+categories          security
+license             GPL-3+
+platforms           any
+maintainers         nomaintainer
+description         ACMEv2 client written in plain C with minimal dependencies
+long_description    lightweight client for the RFC8555 ACMEv2 protocol, \
+                    written in plain C with minimal dependencies \
+                    (libcurl and OpenSSL).
+homepage            https://github.com/ndilieto/uacme
+
+github.setup        ndilieto uacme 1.7.4 v
```

Comment:
`github.setup` implicitly sets `name`, `version`, and `homepage`.  You can move `github.setup` to the top of the line, and delete the `name`, `version` and `homepage` lines.